$OpenBSD: patch-panels_display_meson_build.orig,v 1.1 2021/10/30 09:14:55 ajacoutot Exp $

No colord (brings GUdev and GUsb) on OpenBSD (XXX add HAVE_COLORD to meson_options.txt)

Index: panels/display/meson.build
--- panels/display/meson.build.orig
+++ panels/display/meson.build
@@ -44,7 +44,6 @@ sources += gnome.compile_resources(
 )
 
 deps = common_deps + [
-  colord_dep,
   gnome_desktop_dep,
   m_dep,
   upower_glib_dep
